assets/_sass/abstracts/_variables.scss

```diff
@@ -41,6 +41,7 @@ $size_sidebar-width: 40%;
 $size_post-content-border-top: 6px;
 $size_home-unit-size: 6vw;
 $size_home-unit-size-desktop: 3vw;
+$size_boeing-container-max-width: 700px;
```

Comment:
This is a good variable name, but this shouldn't go in the site's overall `_variables` file. Instead, it should go at the top of the `/boeing/_base.scss` file because that's what it is relevant to. It won't be used out side of this interactive, so we don't want to group it with the larger site structure variables.